### PR TITLE
[UNO-399] Fix: error when there are no resources to show

### PR DESCRIPTION
**Pull request by SergiiTao**
Ticket: https://oat-sa.atlassian.net/browse/UNO-399

If thee didn't contain instances the error appeared.

### CODE REVIEW

File: models/classes/creator/TreeItemLookup.php [hunk 1]
```diff
@@ -91,6 +91,10 @@ private function filterTreeData(array $treeData): array
             }
         }
 
+        if (empty($nodeIds)) {
```
Comment (wazelin) [inline models/classes/creator/TreeItemLookup.php:94]: Could you please apply a similar condition to `ListItemLookup` as well?

```
--- a/models/classes/creator/ListItemLookup.php
+++ b/models/classes/creator/ListItemLookup.php
@@ -66,6 +66,10 @@ class ListItemLookup extends ConfigurableService implements ItemLookup
             $limit
         );
 
+        if (empty($result['nodes'])) {
+            return $result;
+        }
+
         $nodeIds = array_map(
             static function (array $node): string {
                 return $node['uri'];
```

File: models/classes/creator/ListItemLookup.php [hunk 2]
```diff
@@ -73,6 +71,10 @@ static function (array $node): string {
             $result['nodes']
         );
 
+        if (empty($result['nodes'])) {
+            return $result;
+        }
+
```
Comment (wazelin) [inline models/classes/creator/ListItemLookup.php:77]: This is better moved above the `array_map` call.

File: models/classes/creator/TreeItemLookup.php (hunk 1 above)
Comment (SergiiTao) [reply models/classes/creator/TreeItemLookup.php:94]: it is a bit more rare case. but sure. thanks

File: models/classes/creator/ListItemLookup.php (hunk 2 above)
Comment (SergiiTao) [reply models/classes/creator/ListItemLookup.php]: it is true

**Pull request merged**